### ISSUE TITLE
V menu DJ se nezobrazuje možnost změny PIAN importem

### ISSUE THREAD

**Issue by motyc** [ARUP-CAS/aiscr-webamcr#1738]
Vyděleno z #379

- [ ] _pian_import_change_ - http://192.168.254.30:8080/arch-z/akce/detail/M-9101236A/dj/M-9101236A-D01 - v menu (...) se nezobrazuje; nevím ale, zda je to problém oprávnění (#379), nebo úprav v rámci #679

**Comment by pesikj** [comment]
@motyc Já tam tu možnost s admin účtem vidím. Pokud ji tam nevidíte, tak to asi bude dáno oprávněními.

![image](https://github.com/ARUP-CAS/aiscr-webamcr/assets/15179154/a2cac0bc-d9e9-44ae-bfc7-9024efa0fa47)

**Comment by motyc** [comment]
@pesikj Ano, souhlas. Dám to pod oprávnění. Teď jsem to zkoušel, a bude tam někde chybně fixně nastavená vazba na stav záznamu.
